Workbooks/TrendMicroXDROverview.json

```diff
@@ -0,0 +1,127 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## Trend Micro XDR\n---\n[Trend Micro Sucuess Portal](https://success.trendmicro.com)\n\nIf you have any feedback, please send to <mailto:xdrfeedback@trendmicro.com>"
+      },
+      "name": "text - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "TrendMicro_XDR_CL\n| summarize Alerts = count(modelSeverity_s) by bin(TimeGenerated,1d), modelSeverity_s\n| where TimeGenerated >= ago(7d)\n| render columnchart kind=stacked",
+        "size": 0,
+        "title": "WorkBench Alerts Last 7 days",
+        "timeContext": {
+          "durationMs": 2592000000
+        },
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
```

Comment:
A specific subscription id should be referenced from workbook template.  I think a query language needs to be used: https://azure.microsoft.com/en-us/blog/query-across-resources/

@morshabi, @preetikr,  could you help to provide a more specific instruction on how to fix this?